src/consent.ts

```diff
@@ -303,8 +457,11 @@ export default function Consent(mpInstance) {
          * @return {Object} A GDPR Consent State
          */
 
-        function getGDPRConsentState() {
-            return mpInstance._Helpers.extend({}, gdpr);
+        function getGDPRConsentState(): GDPRConsentState {
+            // TODO: Can we replace the usage of extend and either use
```

Comment:
yes i think so

Reply:
assuming there are no subtle gotchas with `mpInstance._Helpers.extend`

Reply:
yes we can.  Helpers.extend should just have been a polyfill, so now that this is being migrated to TS, Object.assign works